test/brod_client_SUITE.erl

```diff
@@ -125,6 +126,19 @@ t_get_partitions_count_safe(Config) when is_list(Config) ->
   ?assertMatch({error, unknown_topic_or_partition}, Res2),
   ok = brod:stop_client(Client).
 
+
+t_get_partitions_count_configure_cache_ttl(Config) when is_list(Config) ->
+  Client = ?FUNCTION_NAME,
+  ClientConfig = [{unknown_topic_cache_ttl, 0}],
```

Comment:
maybe test with 1 or 2, and inspect brod_client:lookup_partitions_count_cache changes from first call `{error, _}` to `false` after a 1/2s sleep.

Reply:
I changed configuration to 100ms and sleep for 101 ms to not slow down tests